kickoff/test/js/tests.js

```diff
@@ -295,6 +295,33 @@ QUnit.test('getPreviousBuildL10nUrl()', function(assert) {
     });
 });
 
+QUnit.test('getUrlAndMessages()', function(assert) {
+    var data = [{
+        product: 'firefox', version: '32.0', buildNumber: 1,
+        // We get file:/// because tests run under grunt-qunit
```

Comment:
this looks copied from previous test but the expected url you use is from https and l10n.m.o. Should this be changed?